build-scripts/createWelcomePage.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
```

Comment:
This (and other build scripts, in a separate change) needs a copyright header.

We should also probably convert our build-scripts to typescript, but that's out of scope for this change.

Reply:
Let's just add a separate fast-follow commit with the headers. That way, we don't have to worry about this potentially becoming inconsistent.